tests/components/uvc/test_camera.py

```diff
@@ -199,6 +220,10 @@ def test_properties(self):
         assert self.uvc.is_recording
         assert "Ubiquiti" == self.uvc.brand
         assert "UVC Fake" == self.uvc.model
+        assert SUPPORT_STREAM == self.uvc.supported_features
+
+    async def test_stream(self):
+        assert "rtsp://host-a:7447/uuid_rtspchannel_0" == await self.uvc.stream_source()
```

Comment:
Missing docstring is causing the Lint error ✌

Reply:
🤦‍♂  I always forget to run that on the tests directory.

Reply:
Me too 😅